roachtest: follower-reads/nodes=3 failed

[(roachtest).follower-reads/nodes=3 failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2168273&tab=buildLog) on [release-19.1@ad8e561d46993a1f4faab1728986ed7f1adff368](https://github.com/cockroachdb/cockroach/commits/ad8e561d46993a1f4faab1728986ed7f1adff368):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/follower-reads/nodes=3/run_1
	follower_reads.go:196,test_runner.go:754: error verifying node values: pq: AS OF SYSTEM TIME: only constant expressions or experimental_follower_read_timestamp are allowed

```

<details><summary>More</summary><p>


Artifacts: [/follower-reads/nodes=3](https://teamcity.cockroachdb.com/viewLog.html?buildId=2168273&tab=artifacts#/follower-reads/nodes=3)
Related:
- #51709 roachtest: follower-reads/nodes=3 failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202007220233_v20.2.0-alpha.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202007220233_v20.2.0-alpha.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50262 roachtest: follower-reads/nodes=3 failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50135 roachtest: follower-reads/nodes=3 failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006032224_v20.2.0-alpha.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006032224_v20.2.0-alpha.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Afollower-reads%2Fnodes%3D3.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).follower-reads/nodes=3 failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2180661&tab=buildLog) on [release-19.1@cebc92d5ad596d721382df9ef18fac2ded589547](https://github.com/cockroachdb/cockroach/commits/cebc92d5ad596d721382df9ef18fac2ded589547):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/follower-reads/nodes=3/run_1
	follower_reads.go:196,test_runner.go:754: error verifying node values: pq: AS OF SYSTEM TIME: only constant expressions or experimental_follower_read_timestamp are allowed

```

<details><summary>More</summary><p>


Artifacts: [/follower-reads/nodes=3](https://teamcity.cockroachdb.com/viewLog.html?buildId=2180661&tab=artifacts#/follower-reads/nodes=3)
Related:
- #52615 roachtest: follower-reads/nodes=3 failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #52614 roachtest: follower-reads/nodes=3 failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #51709 roachtest: follower-reads/nodes=3 failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202007220233_v20.2.0-alpha.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202007220233_v20.2.0-alpha.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50262 roachtest: follower-reads/nodes=3 failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50135 roachtest: follower-reads/nodes=3 failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006032224_v20.2.0-alpha.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006032224_v20.2.0-alpha.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Afollower-reads%2Fnodes%3D3.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cmd/roachtest/follower_reads.go
+++ b/pkg/cmd/roachtest/follower_reads.go
@@ -116,7 +116,7 @@ func runFollowerReadsTest(ctx context.Context, t *test, c *cluster) {
 		return func() error {
 			nodeDB := conns[node-1]
 			r := nodeDB.QueryRowContext(ctx, "SELECT v FROM test.test AS OF SYSTEM "+
-				"TIME follower_read_timestamp() WHERE k = $1", k)
+				"TIME experimental_follower_read_timestamp() WHERE k = $1", k)
 			var got int64
 			if err := r.Scan(&got); err != nil {
 				// Ignore errors due to cancellation.

